app/controllers/org_admin/questions_controller.rb

```diff
@@ -98,18 +98,51 @@ def create
     def update
       question = Question.find(params[:id])
       authorize question
+
+      # get a map from option number to id
+      old_number_to_id = {}
+      question.question_options.each do |opt|
+        old_number_to_id[opt.number] = opt.id
+      end
+      
+      # get a map from question versionable id to old id
+      old_question_ids = {}
+      question.template.questions.each do |q|
+        old_question_ids[q.versionable_id] = q.id
+      end
+
       question = get_modifiable(question)
+
+      # params now out of sync (after versioning) with the question_options
```

Comment:
so are these steps not necessary if the template has not been versioned?
you can determine whether or not the template will be versioned by the `get_modifiable` call above by checking `question.template.generate_version?` which just does a check on its published state

Reply:
looks like you might need to rebase against dev as well